content/initial-subgroup-pr.js

```diff
@@ -0,0 +1,144 @@
+const _ = require('lodash')
```

Comment:
could we re-use the initial-pr.js file, so we don’t have too many duplicate texts?

If too complicated, I’d be okay with the duplication.

Reply:
done